src/wordladder.cpp

```diff
@@ -66,25 +40,31 @@ int main() {
     cout << "first into the second by modifying one letter at a time" << endl;
     cout << endl;
 
-    //Ask for the dictionary file name
+    // Ask for the dictionary file name
+
     getDictionary(dictionary);
     cout << endl;
 
-    //Repeat the following sequence
+    // Repeat the following sequence
+
     while (true) {
         // Ask for word 1 and quit the program if it is blank
         string wordOne = getWord("Word 1 (or Enter to quit): ");
-        if (emptyWord(wordOne)) {
+        if (wordOne == "") {
+            cout << "Exiting... see you later babes!" << endl;
+            pause(2000);

```

Comment:
Since you have these 2 lines duplicated below, move these 2 lines outside of the loop to right before the `cout << "Have a nice day." << endl;` line.

Reply:
Oh yeah! Thanks, addressed.